sympy/matrices/expressions/tests/test_blockmatrix.py

```diff
@@ -97,12 +97,12 @@ def test_BlockMatrix():
 
 
 def test_BlockMatrix_trace():
-    A, B, C, D = map(lambda s: MatrixSymbol(s, 3, 3), 'ABCD')
+    A, B, C, D = [MatrixSymbol(s, 3, 3) for s in 'ABCD']
```

Comment:
Actually I'd simply write this as

```
A = MatrixSymbol('A', 3, 3)
B = MatrixSymbol('B', 3, 3)
C = MatrixSymbol('C', 3, 3)
D = MatrixSymbol('D', 3, 3)
```

The map call is just being clever for the sake of cleverness.
(This does not make your replacement invalid at all, it's just whether it can be improved beyond that - feel free to restrict your mission to fixing one issue at a time, there are good reasons to do that.)